Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/webapp/WebAppNode.java

```diff
@@ -132,129 +131,37 @@ public WebApp getWebapp() {
         return webapp;
     }
 
-    // Delete action class
-    private class DeleteAction extends NodeActionListener implements ActionBackgroundable, ActionPromptable, ActionTelemetrable {
-
-        @Override
-        protected void actionPerformed(NodeActionEvent e) {
-            WebAppNode.this.getParent().removeNode(WebAppNode.this.getSubscriptionId(), WebAppNode.this.getId(), WebAppNode.this);
-        }
-
-        @Override
-        public String getPromptMessage() {
-            return Node.getPromptMessage(AzureActionEnum.DELETE.getName().toLowerCase(), WebAppModule.MODULE_NAME, WebAppNode.this.name);
-        }
-
-        @Override
-        public String getProgressMessage() {
-            return Node.getProgressMessage(AzureActionEnum.DELETE.getDoingName(), WebAppModule.MODULE_NAME, WebAppNode.this.name);
-        }
-
-        @Override
-        public TelemetryParameter getTelemetryParameter() {
-            return TelemetryParameter.WebApp.DELETE;
-        }
+    @AzureOperation(value = "delete web app", type = AzureOperation.Type.ACTION)
+    private void delete() {
+        WebAppNode.this.getParent().removeNode(WebAppNode.this.getSubscriptionId(), WebAppNode.this.getId(), WebAppNode.this);
     }
 
-    // Start action class
-    private class StartAction extends NodeActionListener implements ActionBackgroundable, ActionTelemetrable {
-
-        @AzureOperation(value = "start web app", type = AzureOperation.Type.ACTION)
-        @Override
-        protected void actionPerformed(NodeActionEvent e) {
-            AzureWebAppMvpModel.getInstance().startWebApp(WebAppNode.this.subscriptionId, WebAppNode.this.webapp.id());
-            WebAppNode.this.renderNode(WebAppBaseState.RUNNING);
-        }
-
-        @Override
-        public String getProgressMessage() {
-            return Node.getProgressMessage(AzureActionEnum.START.getDoingName(), WebAppModule.MODULE_NAME, WebAppNode.this.name);
-        }
-
-        @Override
-        public TelemetryParameter getTelemetryParameter() {
-            return TelemetryParameter.WebApp.START;
-        }
+    @AzureOperation(value = "start web app", type = AzureOperation.Type.ACTION)
```

Comment:
I don't like the name 'value'

Reply:
operation?

Reply:
@wangmingliang-ms please adjust it in your PR.